css/media.css

```diff
@@ -0,0 +1,321 @@
+@media(min-width: 320px) {
+    body {
+        box-sizing: border-box;
+        font-family: Montserrat, sans-serif;
+        background-color: #fff;
+    }
+    .header {
+        display: grid;
+        grid-template-columns: 4fr 4fr 4fr;
+        align-items: center;
+        justify-content: center;
+        margin-bottom: 30px;
+    }
+    .header-nav {
+        visibility: hidden;
+    }
+    .header-menu__link {
+        visibility: hidden;
+    }
+    .header-menu {
+        display: flex;
+        justify-content: flex-end;
+        align-items: center;
+    }
+    .header-menu__mobule {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tesla_car/pullRequest?prid=5929182)